generators/languages/templates/src/main/webapp/i18n/bg/global.json.ejs

```diff
@@ -0,0 +1,163 @@
+                "tracker": "Тракер на потребителите",
+                "metrics": "Метрики",
+                "health": "Състояние",
+                "configuration": "Настройки",
+                "logs": "Логове",
+                <%_ if ((databaseType !== 'no' || authenticationType === 'uaa') && databaseType !== 'cassandra') { _%>
+                "audits": "Одити",
+                <%_ } _%>
+                "apidocs": "API",
+                "database": "База данни",
+                "jhipster-needle-menu-add-admin-element": "JHipster will add additional menu entries here (do not translate!)"
+            },
+            "language": "Език"
+        },
+        "form": {
+            "username.label": "Потребителско име",
+            "username.placeholder": "Вашето потребителско име",
+            "currentpassword.label": "Текуща парола",
+            "currentpassword.placeholder": "Текуща парола",
+            "newpassword.label": "Нова парола",
+            "newpassword.placeholder": "Нова парола",
+            "confirmpassword.label": "Потвърждение на новата парола",
+            "confirmpassword.placeholder": "Потвърждение на новата парола",
+            "email.label": "Имейл",
+            "email.placeholder": "Вашият имейл"
+        },
+        "messages": {
+            "info": {
+                "authenticated": {
+                    "prefix": "Ако желаете да ",
+                    "link": "влезете в системата",
+                    "suffix": ", Вие можете да използвате създадените вече акаунти:<br/>- Администратор (потребителско име=\"admin\" и парола=\"admin\") <br/>- Потребител (потребителско име=\"user\" и парола=\"user\")."
+                },
+                "register": {
+                    "noaccount": "Нямате ли все още акаунт?",
+                    "link": "Регистрирайте нов акаунт"
+                }
+            },
+            "error": {
+                "dontmatch": "Паролата и нейното потвърждение не съвпадат!"
+            },
+            "validate": {
+                "newpassword": {
+                    "required": "Вашата парола е задължителна.",
+                    "minlength": "Вашата парола трябва да съдържа поне 4 символа.",
+                    "maxlength": "Вашата парола не може да бъде по-дълга от 50 символа.",
+                    "strength": "Сложност на паролата:"
+                },
+                "confirmpassword": {
+                    "required": "Полето за потвърждаване на вашата парола е задължително.",
+                    "minlength": "Полето за потвърждаване на вашата парола трябва да съдържа поне 4 символа.",
+                    "maxlength": "Полето за потвърждаване на вашата парола не може да бъде по-дълго от 50 символа."
+                },
+                "email": {
+                    "required": "Вашият имейл е задължителен.",
+                    "invalid": "Вашият имейл е невалиден.",
+                    "minlength": "Вашият имейл трябва да съдържа поне 5 символа.",
```

Comment:
Not related to the BG translation: I think it would be better if `5` and `50` are parameters, e.g. `{{min}}` and `{{max}}`

Reply:
Hm. Maybe this won't be that easy. I guess the DB fields have the same length too.

Reply:
I guess someone has to provide "min" and "max parameters as well ;-)